spec/requests/v1/interventions/index_spec.rb

```diff
@@ -200,4 +200,40 @@
       expect(json_response['data'].pluck('id')).to not_include(shared_intervention.id).and not_include(intervention_another_researcher.id)
     end
   end
+
+  context 'when some interventions are starred' do
+    let(:other_researcher) { create(:user, :confirmed, :researcher) }
```

Comment:
I have an idea, what if you create a list of interventions using `create_list` and just one with start, and check if this intervention will be first? Of course, for multiple options, for example: 
- for admin when stared intervention isn't by him -> default order by timestamp 
- admin should see all interventions even if he didn't mark them 
- when you have multiple stared interventions you should get them ordered by timestamp 
- ...